flink-connectors/flink-hbase/src/main/java/org/apache/flink/addons/hbase/HBaseRowInputFormat.java

```diff
@@ -53,78 +47,33 @@
 	private static final long serialVersionUID = 1L;
 
 	private static final Logger LOG = LoggerFactory.getLogger(HBaseRowInputFormat.class);
-	private String tableName;
-	private transient org.apache.hadoop.conf.Configuration conf;
-	private HBaseTableSchema schema;
-	private transient Charset stringCharset;
 
-	// family keys
-	private byte[][] families;
-	// qualifier keys
-	private byte[][][] qualifiers;
-	// qualifier types
-	private int[][] types;
+	private final String tableName;
+	private final HBaseTableSchema schema;
 
-	// row which is returned
-	private Row resultRow;
-	// nested family rows
-	private Row[] familyRows;
+	private transient org.apache.hadoop.conf.Configuration conf;
+	private transient HBaseReadHelper readHelper;
 
 	public HBaseRowInputFormat(org.apache.hadoop.conf.Configuration conf, String tableName, HBaseTableSchema schema) {
```

Comment:
Seems no need to be public.

Reply:
`HBaseRowInputFormat` is a public API to let users construct an `InputFormat` for HBase. They can be used in DataStream or DataSet. 

And it was public before.